_data/sites.json

```diff
@@ -7275,6 +7275,14 @@
         ]
     },
 
+    {
+        "name": "Minds",
+        "url": "https://www.minds.com/settings/other/delete-account",
+        "difficulty": "easy",
+        "domains": [
+            "minds.com"
+        ]
+    },
```

Comment:
Add an empty line after this one please, I'm trying to, but Web interface is not allowing